<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sort_buttons</class>
 <widget class="QWidget" name="sort_buttons">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>138</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>138</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>138</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item alignment="Qt::AlignBottom">
      <widget class="QPushButton" name="date_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>23</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
  background-color: rgb(58, 59, 61);
  color: rgb(101,201,117);
  font-family: -apple-system,BlinkMacSystemFont,&quot;Malgun Gothic&quot;,&quot;맑은고딕&quot;,helvetica,&quot;Apple SD Gothic Neo&quot;,sans-serif;
  font-size: 15px;
  font-weight: bold;
  border-radius: 10px;
}

QPushButton:hover {
  background-color: rgb(89, 91, 94);
}

QPushButton:pressed {
  background-color: rgb(58, 59, 61);
}</string>
       </property>
       <property name="text">
        <string>기간 선택</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
      <widget class="CalOptionWidget" name="cal_option_widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>152</width>
         <height>115</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>152</width>
         <height>115</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignBottom">
      <widget class="QPushButton" name="default_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>23</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
  background-color: rgb(58, 59, 61);
  color: rgb(101,201,117);
  border-radius: 10px;
  font-family: -apple-system,BlinkMacSystemFont,&quot;Malgun Gothic&quot;,&quot;맑은고딕&quot;,helvetica,&quot;Apple SD Gothic Neo&quot;,sans-serif;
  font-size: 15px;
  font-weight: bold;
}

QPushButton:hover {
  background-color: rgb(89, 91, 94);
}

QPushButton:pressed {
  background-color: rgb(58, 59, 61);
}</string>
       </property>
       <property name="text">
        <string>초기화</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="sort_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
        <widget class="QPushButton" name="sort_btn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>23</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  background-color: rgb(58, 59, 61);
  color: rgb(101,201,117);
  border-radius: 10px;
  font-family: -apple-system,BlinkMacSystemFont,&quot;Malgun Gothic&quot;,&quot;맑은고딕&quot;,helvetica,&quot;Apple SD Gothic Neo&quot;,sans-serif;
  font-size: 15px;
  font-weight: bold;
}

QPushButton:hover {
  background-color: rgb(89, 91, 94);
}

QPushButton:pressed {
  background-color: rgb(58, 59, 61);
}</string>
         </property>
         <property name="text">
          <string>최신순</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CalOptionWidget</class>
   <extends>QWidget</extends>
   <header>cal_option_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
